wercker.yml

```diff
@@ -30,8 +30,8 @@ build:
     - script:
         name: Setup newer gcc toolchain and libraries
         code: |
-          cd /etc/yum.repos.d && wget http://people.centos.org/tru/devtools-2/devtools-2.repo
-          yum install -y devtoolset-2-gcc devtoolset-2-binutils devtoolset-2-gcc-c++
+            cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/hhorak/devtoolset-4-rebuild-bootstrap2/repo/epel-6/hhorak-devtoolset-4-rebuild-bootstrap2-epel-6.repo
```

Comment:
We don't have to use devtoolset-4.
Because default toolchain is based on gcc 4.8 in CentOS 7.
(This is still required for CentOS 6, but we'll not use it anymore.)